pyproject.toml

```
@@ -14,6 +14,8 @@ dependencies = [
   "asdf-astropy>=0.6.0",
   "numpy>=1.25",
   "astropy>=6.1",
+  "gwcs @ git+https://github.com/spacetelescope/gwcs.git@master",
+  # "gwcs>=0.26.0",
```

```suggestion
  "gwcs @ git+https://github.com/spacetelescope/gwcs.git@master",
  # "gwcs>=0.26.0",
```
What do we need to do to remove this dev pin?

---

Thanks. @nden when is the next release expected?
I think we should delay this PR until a gwcs release to avoid the dev pin in main. These cause too many headaches.

---

Thanks. Let's delay in this case. I'm happy to discuss reasoning outside this PR if it's helpful.